packages/odyssey-react/src/components/Tag/index.tsx

```diff
@@ -10,26 +10,27 @@
  * See the License for the specific language governing permissions and limitations under the License.
  */
 
-import React from 'react'
-import type { FunctionComponent } from 'react'
+import React from 'react';
+import type { FunctionComponent } from 'react';
+import styles from './Tag.module.scss';
```

Comment:
Since we are embedded tag related styles here .... do we need to remove the tag component styles to be loaded in preview.scss for storybook ?

Reply:
Hmmm good thought. We don't need to as the styles won't be applied into the DOM if the classes are never referenced but it is a good exercise in completeness. [I tried a number of approaches for this](https://okta.slack.com/archives/GSALTH1B6/p1628002238011100) using just build configuration without success. The alternative would of course be to restructure the SCSS imports themselves to be more granular and then excluding Tag styles. Something I'd like to tackle in a downstream PR !

Reply:
@rajaguruduraisamy-okta I ultimately included each scss partial in isolation here so we can start removing them one by one. SCSS partials don't align 1:1 with components as there are contextual styles that cross between boundaries or multiple components listed within on partial but we'll make an effort to keep removing them here. Good thought